src/test/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMergerTest.java

```diff
@@ -114,163 +111,4 @@ public void noTableOfContentsBundleTitleFrequencyTest() throws IOException {
         int expectedBundleTitleFrequency = firstDocBundleTitleFrequency + secondDocBundleTitleFrequency;
         assertEquals(stitchedDocBundleTitleFrequency, expectedBundleTitleFrequency);
     }
-
-
-    //  Folder Coversheets stuff //
```

Comment:
These all got moved to a separate file, because they're testing coversheets specifically, and because this way the other stuff gets an @before method